responsibleai_vision/responsibleai_vision/utils/image_reader.py

```diff
@@ -90,9 +90,9 @@ def get_all_exif_feature_names(image_dataset):
                 for tag_id in exifdata:
                     # get the tag name, instead of human unreadable tag id
                     tag = TAGS.get(tag_id, tag_id)
-                    # TODO: add support for other data types
-                    if tag not in image_dataset.columns \
-                       and isinstance(exifdata.get(tag_id), str):
+                    if tag not in image_dataset.columns:
+                       data = exifdata.get(tag_id)
+                       if isinstance(data, str) or isinstance(data, int) or isinstance(data, float):
                         exif_feature_names.add(tag)
```

Comment:
Can you fix the spacing here? it looks like there is a single space on line 96.  Does this actually work in python?
Oh, I guess python accepts 1, 2, or 4 spaces according to this:
https://stackoverflow.com/questions/2034517/pythons-interpretation-of-tabs-and-spaces-to-indent
Still I think we need the same number of spaces for indent.  Maybe we need to have some linting for this.

Reply:
Yes plan to fix linting at the end after all technical comments are resolved :)